audio/src/main/res/layout/exo_playback_control_view.xml

```diff
@@ -0,0 +1,102 @@
+<?xml version="1.0" encoding="utf-8"?><!-- Copyright (C) 2016 The Android Open Source Project
+
+     Licensed under the Apache License, Version 2.0 (the "License");
+     you may not use this file except in compliance with the License.
+     You may obtain a copy of the License at
+
+          http://www.apache.org/licenses/LICENSE-2.0
+
+     Unless required by applicable law or agreed to in writing, software
+     distributed under the License is distributed on an "AS IS" BASIS,
+     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+     See the License for the specific language governing permissions and
+     limitations under the License.
+-->
+<android.support.constraint.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@color/audioBackground"
+    android:paddingStart="16dp"
+    android:paddingEnd="16dp"
+    android:paddingBottom="26dp">
+
+    <TextView
+        android:id="@id/exo_position"
+        style="@style/MetadataMediumGray"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:fontFamily="@font/ideal_sans_book"
+        android:includeFontPadding="false"
+        android:paddingEnd="2dp"
+        android:textSize="14sp"
```

Comment:
zeplin styles please

Reply:
It is for the video's `current time` field. There's no style for this specific component since it is displayed as one unit in zeplin (``e.g. 1:30/3:00``). I just applied the value which looked similar to the design. I moved the dimension to the values file.